src/main/java/com/shopify/minishopify/controllers/ShopController.java

```diff
@@ -61,4 +61,14 @@ public Shop updateShop(@Valid @RequestBody Shop updatedShop, @PathVariable int i
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, String.format("No shop found with id: %d", id));
         }
     }
+
+    @GetMapping("/shopList")
```

Comment:
Change to just "/"

Reply:
yeah won't work with /
i'll change it to /list